packages/example/src/components/Transfer/Transfer.tsx

```diff
@@ -1,21 +1,53 @@
-import React from "react";
+import React, {useState} from "react";
 import {Box, Button, Card, CardContent, CardHeader, Grid, TextField} from '@material-ui/core/';
+import {getInjectedMetamaskExtension} from "../../services/metamask";
+
+interface ITransferProps {
+    network: string
+}
+
+export const Transfer: React.FC<ITransferProps> = ({network}) => {
+    const [recipient, setRecipient] = useState<string>("");
+    const [amount, setAmount] = useState<string | number>("");
+
+    const handleCurrency = (network: string): string => {
+        switch(network) {
+            case "kusama": return "KSM";
+            case "westend": return "WST";
+            default: return ""
+        }
+    }
+
+    const handleRecipientChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setRecipient(event.target.value);
+      };
+    const handleAmountChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setAmount(event.target.value);
+    };
```

Comment:
Even though it probably doesn't matter in current case, lets use best practices, this should be
useCallback with setAmount as dependency:
https://reactjs.org/docs/hooks-reference.html#usecallback

Same for handleRecipientChange